lib/multibuild/registry-secrets.ts

```diff
@@ -23,9 +23,12 @@
 
 import * as ajv from 'ajv';
 
+import { getRegistryAndName } from './utils';
 import { RegistrySecretValidationError } from './errors';
 export { RegistrySecretValidationError } from './errors';
 
+
+
```

Comment:
Blank lines, you might need to run prettier if it didn’t run automatically on commit.